### ISSUE TITLE
Unable to verify linux binary for operator-sdk v1.0.0

### ISSUE THREAD

**Issue by demonCoder95** [operator-framework/operator-sdk#3743]
## Bug Report

**What did you do?**
I was following the [Installation Guide on Website](https://sdk.operatorframework.io/docs/installation/install-operator-sdk/) to install the linux binary for operator-sdk v1.0.0.
When I got to the step of verifying the downloaded binary, I got the following output

```bash
gpg: assuming signed data in 'operator-sdk-v1.0.0-x86_64-linux-gnu'
gpg: Signature made Mon Aug 10 19:15:08 2020 EDT
gpg:                using DSA key BF6F6F18846753754CBB1DDFBC9679ED89ED8983
gpg: Can't check signature: No public key
```
Where I assumed ``BF6F6F18846753754CBB1DDFBC9679ED89ED8983`` is the ``$KEY_ID`` and tried to get the key from the Ubuntu keyserver

```bash
$ gpg --keyserver keyserver.ubuntu.com --recv-key "BF6F6F18846753754CBB1DDFBC9679ED89ED8983"
```
**What did you expect to see?**
A successful key download for binary verification

**What did you see instead? Under which circumstances?**

I got the following output
```bash
gpg: keyserver receive failed: No keyserver available
```

**Environment**
These details I got after running the binary without verifying it. I haven't setup Golang/K8s or anything else yet.

``operator-sdk version: "v1.0.0", commit: "d7d5e0cd6cf5468bb66e0849f08fda5bf557f4fa", kubernetes version: "v1.18.2", go version: "go1.13.11 linux/amd64", GOOS: "linux", GOARCH: "amd64"``

**Possible Solution**
I'm thinking this has something to do with the binary signed with some unknown key? Maybe someone can look into this. Or perhaps the keyserver needs to have the key stored on it

**Additional context**
Nil

**Comment by demonCoder95** [comment]
I'm happy to fix this with a PR given the right context. I'll keep trying to fix it.

**Comment by demonCoder95** [comment]
If I try without the keyserver, I get the following output
```bash
gpg --recv-key "BF6F6F18846753754CBB1DDFBC9679ED89ED8983"
gpg: keyserver receive failed: General error
```

**Comment by camilamacedo86** [comment]
Hi @jmrodri, 

when you build the binaries were your signature setup? More info [here](https://sdk.operatorframework.io/docs/contribution-guidelines/release/#binaries-and-signatures). 

Hi @demonCoder95, 
could you please help us and check the version 0.19.2 as well?

**Comment by jmrodri** [comment]
@demonCoder95 that is definitely my key. I recognize it. Based on the above sounds like the keyserver was having issues because when I run the same command:
```
$ gpg --keyserver keyserver.ubuntu.com --recv-key "BF6F6F18846753754CBB1DDFBC9679ED89ED8983"
gpg: /home/liveuser/.gnupg/trustdb.gpg: trustdb created
gpg: key BC9679ED89ED8983: public key "Jesus Miguel Rodriguez (jesusr) <jesusr@redhat.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```
I ran this from a livecd of Fedora so it did not have any of my keys locally.

**Comment by camilamacedo86** [comment]
It worked for me as well in the Mac OS X: 

```
 $ gpg --keyserver keyserver.ubuntu.com --recv-key "BF6F6F18846753754CBB1DDFBC9679ED89ED8983"
gpg: key BC9679ED89ED8983: public key "Jesus Miguel Rodriguez (jesusr) <jesusr@redhat.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1

```

**Comment by jmrodri** [comment]
I was able to verify the signatures: 
```
[jesusr@transam foo]$ gpg --verify operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu.asc
gpg: assuming signed data in 'operator-sdk-v1.0.0-x86_64-linux-gnu'
gpg: Signature made Mon 10 Aug 2020 07:15:08 PM EDT
gpg:                using DSA key BF6F6F18846753754CBB1DDFBC9679ED89ED8983
gpg: Good signature from "Jesus Miguel Rodriguez (jesusr) <jesusr@redhat.com>" [ultimate]
gpg:                 aka "jesus rodriguez (jmrodri) <jmrodri@gmail.com>" [ultimate]
[jesusr@transam foo]$ gpg --verify ansible-operator-${RELEASE_VERSION}-x86_64-linux-gnu.asc
gpg: assuming signed data in 'ansible-operator-v1.0.0-x86_64-linux-gnu'
gpg: Signature made Mon 10 Aug 2020 07:15:11 PM EDT
gpg:                using DSA key BF6F6F18846753754CBB1DDFBC9679ED89ED8983
gpg: Good signature from "Jesus Miguel Rodriguez (jesusr) <jesusr@redhat.com>" [ultimate]
gpg:                 aka "jesus rodriguez (jmrodri) <jmrodri@gmail.com>" [ultimate]
[jesusr@transam foo]$ gpg --verify helm-operator-${RELEASE_VERSION}-x86_64-linux-gnu.asc
gpg: assuming signed data in 'helm-operator-v1.0.0-x86_64-linux-gnu'
gpg: Signature made Mon 10 Aug 2020 07:15:13 PM EDT
gpg:                using DSA key BF6F6F18846753754CBB1DDFBC9679ED89ED8983
gpg: Good signature from "Jesus Miguel Rodriguez (jesusr) <jesusr@redhat.com>" [ultimate]
gpg:                 aka "jesus rodriguez (jmrodri) <jmrodri@gmail.com>" [ultimate]
```

**Comment by camilamacedo86** [comment]
So, @jmrodri  I think we can close this one as solved. 🥇

**Comment by demonCoder95** [comment]
Let me try again. Maybe the key server indeed had some issues. @jmrodri 
@camilamacedo86 Sure, I can test v0.19.2 as well.

**Comment by demonCoder95** [comment]
I tried again, with no luck, interestingly, older versions that I've verified before are still causing an issue. I'm trying to do the same on another linux machine now. There's probably something the matter with the keyserver setup on this machine. I'll dig some more into it, maybe this could provide us with a troubleshooting tip for gpg keyservers.

**Comment by demonCoder95** [comment]
I've verified a good signature on another system. I'll troubleshoot the keyserver problem on the ground 0 system and keep you posted. Thank you for the quick response! :)

**Comment by jmrodri** [comment]
Closing this as solved. @demonCoder95 if you have issues again please let us know.
